recipes/python/lxml.yaml

```diff
@@ -0,0 +1,41 @@
+inherit: [patch, python3-cext-pkg]
+
+metaEnvironment:
+    PKG_LICENSE: "BSD-3-Clause"
+    PKG_VERSION: "5.3.1"
+
+depends:
+    - tools:
+          target-toolchain: host-compat-toolchain
+      use: [tools]
+      depends:
+          - libs::libxml2-bin
+          - libs::libxslt-bin
+    - name: python::cython
+      tools:
+          target-toolchain: host-compat-toolchain
+
+    - libs::libxml2-dev
+    - libs::libxslt-dev
+    - use: []
+      depends:
+          - libs::libxml2-tgt
+          - libs::libxslt-tgt
+
+checkoutSCM:
+    scm: url
+    url: https://github.com/lxml/lxml/archive/refs/tags/lxml-${PKG_VERSION}.tar.gz
+    digestSHA1: f2c04d861f72d7119697194f6e46d3cdfd272e0c
+    stripComponents: 1
+
+checkoutDeterministic: True
+checkoutScript: |
+    patchApplySeries $<@lxml/*.patch@>
```

Comment:
The patch is missing?

Reply:
🤦 Apparently it was quite late. Might take some time because I'm afk today.

Reply:
Let's try again...